wazo_calld/plugins/calls/exceptions.py

```diff
@@ -32,6 +32,16 @@ def __init__(self, call_id):
         )
 
 
+class CallOriginUnavailableError(APIException):
+    def __init__(self, call_id):
```

Comment:
Could be valuable to include `line_id` info(could be `None` by default if the error can occur without a specific line id)